Tests/AK/TestTime.cpp

```diff
@@ -274,3 +274,11 @@ TEST_CASE(is_negative)
     EXPECT_EQ(result.to_nanoseconds(), 5);
     EXPECT(!result.is_negative());
 }
+
+TEST_CASE(day_of_week)
+{
+
```

Comment:
Please remove this empty line. Did you run `clang-format`?

Reply:
I thought i did. 

`
bash ./Meta/lint-clang-format.sh --overwrite-inplace $(git diff --name-only HEAD~2..HEAD .)
`

But nothing changes.